cmd/zpool/zpool_main.c

```diff
@@ -6123,8 +6286,11 @@ zpool_do_status(int argc, char **argv)
 			break;
 		case '?':
 			if (optopt == 'c') {
-				fprintf(stderr,
-				    gettext("Missing CMD for -c\n"));
+				fprintf(stderr, gettext(
+				    "Missing script for -c.  Current scripts "
+				    "in %s:\n"),
+				    ZPOOL_SCRIPTS_DIR);
```

Comment:
How about not framing this as an error and just printing the available scripts.  It would be really convenient if we could provide a brief description extracted from the script, maybe by invoking it with a `-h` option.  For example:

```
Available scripts:
  enc       Print SCSI Enclosure Services (SES) info.
  iostat    Display most relevant iostat bandwidth/latency numbers.
  ...
```

 We should probably also restrict the entries returned to file which are executable.